subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AvailableJavaHomes.java

```diff
@@ -188,11 +188,10 @@ public static File getBestJre() {
             NativeServices nativeServices = NativeServicesTestFixture.getInstance();
             FileCanonicalizer fileCanonicalizer = nativeServices.get(FileCanonicalizer.class);
             jvms = new ArrayList<>();
-            jvms.addAll(new DevInfrastructureJvmLocator(fileCanonicalizer).findJvms());
             InstalledJvmLocator installedJvmLocator = new InstalledJvmLocator(OperatingSystem.current(), Jvm.current(), nativeServices.get(WindowsRegistry.class), nativeServices.get(SystemInfo.class), fileCanonicalizer);
             jvms.addAll(installedJvmLocator.findJvms());
             if (OperatingSystem.current().isLinux()) {
-                jvms.addAll(new BaseDirJvmLocator(fileCanonicalizer, new File("/opt")).findJvms());
+                jvms.addAll(new BaseDirJvmLocator(fileCanonicalizer, new File("/opt/jdk")).findJvms());
```

Comment:
Do you think that may be problematic for people using Windows on their development machines? JDKs in `/opt` wouldn't be detected any more, right?

Reply:
Actually I'm confused by the old code, why it was `/opt` here? We have all JDKs install in `/opt/jdk`, so I assumed it was legacy code. Why this is problematic for windows users? Does windows have `/opt`?

We can have a chat on this, I was puzzled a lot by the old code.

Reply:
This is not only about Linux on CI. This is also about Linux on developer machines. I suppose some folks installed Java in `/opt`. If that is not the case then the change is fine.

Reply:
@melix Linux guy! Will this be a problem for you?